.github/workflows/pull_request.yml

```diff
@@ -17,6 +20,11 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+      - name: Checkout PR
+        if: ${{github.actor == 'dependabot[bot]' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: gh pr checkout ${{ github.event.pull_request.number }}
```

Comment:
Could we please move the new actions to a separate "manifest" workflow and change pull_request.yml to run like so?

```
# Trigger when the manifest workflow succeeded
on:
  workflow_run:
    workflows: ["manifest build"]
    types:
      - completed
```

I'm thinking separate workflows would be cleaner because it's a dependabot only feature and easier to disable the "pull_request" workflow, eventually.

Note that we're migrating to the Insights container build pipeline by the end of Q4, so we won't likely use the "pull_request" workflow to build PRs. However, it's possible that we could continue to using a separate "manifest" workflow, while disabling the "pull_request" workflow only.